rdt/hyper_transformer.py

```diff
@@ -207,6 +220,10 @@ def _validate_all_fields_fitted(self):
             warnings.warn('The following fields were specified in the input arguments but not'
                           + f'found in the data: {non_fitted_fields}')
 
+    @staticmethod
+    def _get_null_column_name(field):
+        return ''.join(field.split('.')[:-1]) + '.is_null'
```

Comment:
This does not seem correct: What happens if the columns `something.transformed` and `something_else.transformed` exist? This would return `transformed.is_null` in both cases, which would be wrong? (Note: This scenario should be covered in an integration test)

The null column name should simply be `f{column_name}.is_null}`